learning.py

```diff
@@ -839,10 +839,8 @@ def network(input_units, hidden_layer_sizes, output_units, activation=sigmoid):
     excluding input and output layers
     """
     # Check for PerceptronLearner
```

Comment:
Please remove this comment and the new line below it.

Reply:
@MrDupin  okay . Thank you for review